app/services/acuant/liveness_pass_through.rb

```diff
@@ -0,0 +1,62 @@
+module Acuant
+  class LivenessPassThrough < AcuantBase
+    def call(liveness_body)
```

Comment:
can we document what the return values are? the parts of the tuple? Here's my guess

maybe:

```ruby
@return [Array(Boolean, Boolean)] a tuple of (overall success, face success)
def call(liveness_body)
```

Reply:
Thanks for adding documentation.... the syntax like you have it now in YARD means it returns a single element that is of type "is_photo_live?" or "does_face_match?"... which are not classes.... I'd recommend something similar to what I wrote

Reply:
Actually on second thought, I just came across this ticket we have: https://cm-jira.usa.gov/browse/LG-2961

WDYT of switching this to return a struct with named properties instead?

Reply:
Good ideas Zach.  After thinking about it the service caller will only ever get 3 states good/not live/no facematch.  I think I'm just going to return an enum

Reply:
And I need to log these errors so that we can do analysis afterwards